Fix small issue on line 230 in calculate_complexity_scores.py

jguillen15 opened the pull request:
There is an issue with the if statement on line 230. This statement tests whether "credentials" is a key in the dictionary (i.e., is it an argument), which is always true.  It should instead be checking the value that is assigned to that key.

jguillen15 committed:
Fix small issue on line 230 in calculate_complexity_scores.py

There is an issue with the if statement on line 230. This statement tests whether "credentials" is a key in the dictionary (i.e., is it an argument), which is always true.  It should instead be checking the value that is assigned to that key.

jakebeal commented:
Tests are failing - please identify the problem and fix it so that the tests pass.

jguillen15 committed:
Reverse Update calculate_complexity_scores.py

jguillen15 committed:
Rename iGEM_BBF10K_000***.gb to iGEM_BBF10K_000___.gb

File with original name gives issues when cloning

jguillen15 committed:
Update test_calculate_complexity_scores.py

Avoid reusing p1 == p2 condition in both sides of 
"and" logical function

jguillen15 committed:
Update calculate_complexity_scores.py

There is an issue with the if statement on line 230. This statement tests whether "credentials" is a key in the dictionary (i.e., is it an argument), which is always true.  It should instead be checking the value that is assigned to that key.

jguillen15 committed:
Rename iGEM_BBF10K_000___.gb to iGEM_BBF10K_000***.gb

jguillen15 committed:
Update Comparison_file_Complexity_Scores.nt

Change expected complexity score of first and third sequence. This is due to a change that was noticed in the IDT API.

jakebeal commented:
The tests are succeeding in the source fork. Their failure here appears to be due to GitHub's protection against sharing secrets across forks. Accordingly I'm going to try merging and see if that fixes it...
